lib/main.js

```diff
@@ -115,6 +115,8 @@ const Hydrogen = {
           this.handleKernelCommand({ command: "interrupt-kernel" }),
         "hydrogen:restart-kernel": () =>
           this.handleKernelCommand({ command: "restart-kernel" }),
+        "hydrogen:refresh-all-result-bubbles": () =>
```

Comment:
Thanks for explaining the reasoning behind it :+1:

@rgbkrk @BenRussert What do you think, should we go with `refresh-all-result-bubbles`?